src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +615,33 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Set the event dispatcher instance.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
+     * @return void
+     */
+    public function setEventDispatcher(Dispatcher $dispatcher)
+    {
+        static::$dispatcher = $dispatcher;
+
+        return $this;
+    }
+
+    /**
+     * Fire the given event for the model.
+     *
+     * @param  string  $event
+     * @param  array  $params
+     * @return mixed
+     */
+    public function fireMigrationEvent($name, $params = [])
+    {
+        if (! isset(static::$dispatcher)) {
+            return true;
+        }
+
+        return static::$dispatcher->dispatch("migrations.{$name}", $params);
+    }
```

Comment:
Instead of returning something which isn't needed, make this method void and try this:

```php
	    public function fireMigrationEvent($name, $params = [])
	    {
	        if ($this->events) {
	            $this->events->dispatch("migrations.{$name}", $params);
	        }
	    }
```

Reply:
Done